release-notes.hbs.md

```diff
@@ -151,6 +162,17 @@ This release includes the following changes, listed by component and area.
   about your external secrets and secret stores. The CREATE command lets you create cluster
   external secret and cluster secret stores. For more information, see the [Tanzu CLI Command Reference](https://docs.vmware.com/en/VMware-Tanzu-CLI/1.0/tanzu-cli/command-ref.html) documentation.
 
+#### <a id='1-7-0-service-bindings'></a> v1.7.0 Features: Service Bindings
+
+- Introduces the new `servicebinding.tanzu.vmware.com` package, which supersedes the existing
+  `service-bindings.labs.vmware.com`. The new package is based on the VMware-maintained
```

Comment:
**issue** This is the other way round, the new Package is based on the community-maintained version

Reply:
ah LOL, let me fix it immediately!